app/models/ride.rb

```diff
@@ -1,3 +1,9 @@
 class Ride < ApplicationRecord
   belongs_to :amusement_park
+  has_many :mechanic_rides
+  has_many :mechanics, through: :mechanic_rides
+
+  def mechanic_avg_exp
```

Comment:
Nice job building this helper method to call in your method in the `amusement_park.rb` model.